### PR TITLE
Mini google assistant 

### LINKED ISSUE #726: Mini Google Assistant
# Aim

- To build a mini google assistant.

# Details

- A mini google assistant using some python libraries. Kindly assign me this issue, @avinashkranjan Sir.

### PR DESCRIPTION
**Pull request by A-kriti**
# Description

Added a new folder "Mini Google Assistant" containing python and a readme file which finally fixes issue no. 726.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [✓] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [✓] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [✓] My code follows the style guidelines(Clean Code) of this project
- [✓] I have performed a self-review of my own code
- [✓] I have commented my code, particularly in hard-to-understand areas
- [✓] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [✓] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: mini google assistant.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import speech_recognition as sr
```
[the remark this answers is not in the document]
Comment (A-kriti) [reply mini google assistant.py:1]: Done!

**Review by Sanskrutim17 — COMMENTED** [review]
could add a statement to engine
 like wishing the listener  and asking them how can i  help you
sounds nice

**Comment by A-kriti** [comment]
@Sanskrutim17 It's already there I guess! and do tell if something to add :)

**Comment by kaustubhgupta** [comment]
@Sanskrutim17 are you the mentor of this project?

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Oops, you deleted the wrong file! You had to delete the file outside the folder but you deleted the file inside folder one. Shift the outside file to Mini Google Assistant folder

**Comment by A-kriti** [comment]
@kaustubhgupta Sir, done!

File: mini google assistant.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import pyttsx3
+import pywhatkit
+import datetime
+import pyjokes
+import cv2
```
Comment (kaustubhgupta) [inline mini google assistant.py:6]: now delete this file:

image

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: mini google assistant.py (hunk 2 above)
Comment (A-kriti) [reply mini google assistant.py]: @kaustubhgupta Sir, Done!

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**